### ISSUE TITLE
Format of time input for scheduling events is inconsistent

### ISSUE THREAD

**Issue by mikepmerritt** [mikepmerritt/Challenge-1#16]
When scheduling an event outside the gym, the user is asked to give the time in military time. When scheduling an event in the gym, the user is asked for a time and then if that time is AM or PM. Both implementations work, but choosing one over the other would enhance the user experience.

**Comment by Alex112292** [comment]
Yes for consistency sake this is an important change to make for overall user experience.
